_rules/text-spacing-not-important-b8bb68.md

```diff
@@ -0,0 +1,264 @@
+---
+id: b8bb68
+name: Text spacing in `style` attributes is not `!important`
+rule_type: atomic
+description: |
+  This rule checks that the text spacing styles can be adjusted by user, i.e. that `!important` is not applied to the style, unless the style has already exceeded the specified metric for retaining content visibility and functionality.
+accessibility_requirements:
+  wcag21:1.4.12: # Text Spacing (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Jey Nandakumar
+---
+
+## Applicability
+
+This rule applies to any HTML element that is [visible][] and has a [declared][] value for any of the following CSS properties:
+
+- [word-spacing][],
+- [letter-spacing][],
+- [line-height][];
+  which are set by the `style` attribute.
+
+## Expectation 1
+
+For the test target's [word-spacing][] property, either:
+
+- it has no [declared][] value; or
+- its [declared][] value does not have the [important flag][]; or
```

Comment:
This now runs into the issue I outlined here: https://github.com/act-rules/act-rules.github.io/pull/1251#discussion_r443742146. That is, there may be _multiple_ declared values for a given property, but we're only interested in the one in the `style` attribute. This applies to all three expectations.